src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Endpoint.cs

```diff
@@ -0,0 +1,211 @@
+using System.Data.Common;
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Public.API.Helpers;
+using Dapper;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.Get;
+
+public sealed class Endpoint(AquiferDbContext _dbContext, ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Get("/resources/collections/{code}");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+        Summary(s =>
+        {
+            s.Summary = "Gets a resource collection with language localization data for the given collection code.";
+            s.Description =
+                "Returns the resource collection matching the collection code including the count of resource items in each available language.";
```

Comment:
Is "resource items" the correct language for public API documentation of what we call "resources"?